modules/dcache-srm/src/main/java/diskCacheV111/srm/dcache/SrmReserveSpaceCompanion.java

```diff
@@ -101,7 +101,7 @@ private SrmReserveSpaceCompanion(SrmReserveSpaceCallback callback)
     @Override
     public void failure(int rc, Object error)
     {
-        LOGGER.error("Space Reservation Failed rc:" + rc + " error:" + error);
+        LOGGER.error("Space Reservation Failed rc: {} error:{}", rc, error.toString());
```

Comment:
As it not guaranteed, that error is not **null**, you can get a NullPointerException here. String#valueOf() will be  a safer solution here.